packages/engine/paima-sm/src/index.ts

```diff
@@ -180,7 +180,7 @@ const SM: GameStateMachineInitializer = {
               caip2: '',
               txHash: '',
             },
-            blockHeight,
+            { blockHeight, timestamp: 0 },
```

Comment:
we don't store the timestamps, so there is no easy way of getting the timestamp of the latest processed block. I'm not sure how much this matters though, since I'm not sure how the dry run is supposed to be used.

an easy alternative would be to put the current time maybe, but it's a bit weird.